### PR TITLE
Comment out heart + comment icons

### PR DESCRIPTION
**Pull request by bitterdev**

### CODE REVIEW

**Comment by aembler** [comment]
We should use PHP comments on this or remove it entirely so that it doesn't actually do all the database queries.

**Review by aembler — CHANGES_REQUESTED** [review]
Please see the previous comment.

**Comment by bitterdev** [comment]
@aembler Hi Andrew i have used the php comment function like required.

File: public/packages/concrete_cms_docs/blocks/tutorial_list/view.php [hunk 1]
```diff
@@ -149,13 +149,15 @@ class="btn btn-primary">
                                         ) ?>
                                     </div>
 
+                                    <!--
```
Comment (aembler) [inline public/packages/concrete_cms_docs/blocks/tutorial_list/view.php:152]: I think you missed this one

**Review by aembler — CHANGES_REQUESTED** [review]

File: public/packages/concrete_cms_docs/blocks/tutorial_list/view.php (hunk 1 above)
Comment (bitterdev) [reply public/packages/concrete_cms_docs/blocks/tutorial_list/view.php]: i have fixed it thank you

**Pull request merged**